packages/components/src/DataViewer/theme.scss

```diff
@@ -0,0 +1,56 @@
+$tc-layout-skeleton-width: 9.6rem !default;
+$tc-layout-skeleton-height: 9.6rem !default;
+
+.tc-twoviewers-layout {
+	display: flex;
+	width: 100%;
+	overflow: hidden;
+
+	&-loading {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+
+		:global(.tc-svg-icon) {
+			width: $tc-layout-skeleton-width;
+			height: $tc-layout-skeleton-height;
+		}
+	}
+
+	&-left,
+	&-right {
+		background-color: $wild-sand;
+	}
+
+	&-left {
+		border-right: 0.5rem solid white;
+		width: 47rem;
+	}
+
+	&-right {
+		flex-grow: 1;
+	}
+}
+
+.tdc {
+	:global {
+		@import '~@talend/bootstrap-theme/src/theme/variations/tdc';
```

Comment:
Is it really expected here?

Reply:
Same as above, only used by a story, i don't have rewritten styles to display the components in the story, i reused those that was used in the original story.

Reply:
I don't see any benefits here to get a demo using tdc theme but ok

Reply:
I look to remove it

Reply:
Removed